lib/cycle.js

```diff
@@ -44,138 +156,156 @@ module.exports = function(){
       // store user in && authenticate the session
       req.session.user = user;
       req.session.authenticated = true;
-
       // now respond or redirect
-      var postResponse = this._resolvePostAction(waterlock.config.postActions.login.success, 
+      var postResponse = this._resolvePostAction(waterlock.config.postActions.login.success,
         user);
+      if (postResponse === 'jwt') {
+        //Returns the token immediately
+        var jwtData = waterlock._utils.createJwt(req, res, user);
+
+        Jwt.create({token: jwtData.token, uses: 0, owner: user.id}).exec(function(err){
```

Comment:
I can do this in the am. I need to find the other 3% coverage too so will hit that too

Sent from my iPhone

> On 11 Apr 2015, at 19:54, Jerry Reptak <notifications@github.com> wrote:
> 
> In lib/cycle.js:
> 
> >          user);
> > +      if (postResponse === 'jwt') {
> > +        //Returns the token immediately
> > +        var jwtData = waterlock._utils.createJwt(req, res, user);
> > +
> > +        Jwt.create({token: jwtData.token, uses: 0, owner: user.id}).exec(function(err){
> Maybe create a function for this part (createTokenResponse?) since its used twice in this file (register and login)
> 
> —
> Reply to this email directly or view it on GitHub.
>